samples/boards/sensortile_box/src/main.c

```diff
@@ -248,8 +250,22 @@ static void iis3dhhc_config(struct device *iis3dhhc)
 void main(void)
 {
 	static struct device *led0, *led1;
+	struct device *dev = device_get_binding(
+			CONFIG_UART_CONSOLE_ON_DEV_NAME);
 	int i, on = 1;
 	int cnt = 1;
+	uint32_t dtr;
+
+	/* Application must enable USB by itself */
+	if (usb_enable(NULL)) {
+		return;
+	}
+
+	/* Poll if the DTR flag was set, optional */
+	while (!dtr) {
+		uart_line_ctrl_get(dev, UART_LINE_CTRL_DTR, &dtr);
+		k_msleep(100);
```

Comment:
done

Reply:
k_msleep(100); here is to relax situation and give cpu time for the logging thread.

Reply:
Yes, but this DTR is waiting for console to actually log those so If we put this k_msleep() nothing will be send anyway, because console is not yet active.